src/main/java/org/jabref/gui/preferences/exporter/ExportCustomizationTabViewModel.java

```diff
@@ -23,20 +25,28 @@
     private final DialogService dialogService;
     private final JournalAbbreviationRepository repository;
 
-    public ExportCustomizationDialogViewModel(PreferencesService preferences, DialogService dialogService, JournalAbbreviationRepository repository) {
+    public ExportCustomizationTabViewModel(PreferencesService preferences, DialogService dialogService, JournalAbbreviationRepository repository) {
         this.preferences = preferences;
         this.dialogService = dialogService;
         this.repository = repository;
-        loadExporters();
     }
 
-    private void loadExporters() {
+    @Override
+    public void setValues() {
```

Comment:
For duality, I suggest to rename this as `loadSettings` (or shorter only have `load` and ̀ store` methods).

Reply:
I would like to stick to `setValues` if you don't mind, because I use this naming convention also in other places with parameters that are to be set in a control or to reset the values.
`storeSettings` on the other hand is only used in the PreferencesTabs.